Automattic/wc-calypso-bridge
# Fix/716 wc pay menu icon and notification

Fixes #716 

This PR fixes the following problems.

1. The WC Pay menu icon is not displayed correctly on "My Home" page due to missing CSS.

I've added base64 image, but kept the CSS as well. It was a bit quirky. When I pass `icon` option to `wc_admin_register_page`, it was removed when I go to one of the WC pages. Keeping CSS can fix this problem.

2. The notification badge isn't displayed.

It turned out that the dashboard page expects a bit different notification format than what we used.

New format: `<span class="wcpay-menu-badge awaiting-mod count-1"><span class="plugin-count">1</span></span>`. Notice we have a nested span.

3. It also fixes duplicated WC Pay menu on "My Home" page.

**Note**: You might see overlapped WC Pay menu icon if you test it locally due to having both base64 image and CSS. I haven't found a way to workaround it yet, but I think it's acceptable since this is going to be on .com only.

### Testing instructions

Follow the tutorial in p90Yrv-2j7-p2 to test in WPCOM production. Upload this branch to both business and e-commerce sites and test both.

Confirm you see the correct menu icon and notification badge on "My Home" (dashboard) and WooCommerce -> Home or
